go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/test-network-function/collector
+
+go 1.18
```

Comment:
This should be `go 1.20`, it means your local Go install is 1.18.

Reply:
changed it and will update my local, thanks